front/components/cart/cart_controller.js

```diff
@@ -1,18 +1,9 @@
-angular.module('necs.CartModule.controller', ['AjaxModule'])
-.controller('CartController', ['$scope','ajaxUtil',
-function($scope, ajaxUtil){
+angular.module('vc.CartModule.controller', ['AjaxModule','ProductModule'])
+.controller('CartController', ['$scope','ajaxUtil','productService',
+function($scope, ajaxUtil, productService){
 
-    $scope.products=[
-    {
-      id: 2,
-      name: "maria",
-      description: "",
-      brand: "baby the stars shine bright",
-      price:456,
-      qty:1,
-      image:"/assets/images/products/2.jpg",
-    }
-    ];
+    $scope.products=productService.getProducts();
```

Comment:
fix indentation and spacing.  variableName=value is not as nice as variableName = value